app/lib/CapturePlayer.js

```diff
@@ -1,19 +1,19 @@
 'use strict';
+
 /* eslint-disable no-process-exit, no-sync */
 
 const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
+const url = require('url');
 
 function normalizeRewritePath(_path) {
-  return _path.replace(/[\?\=\&\%\+]+/g, '_').replace(/^\/?/, '/');
+  return _path.replace(/[\?\=\&\%\+]+/g, '_').replace(/^\/?/, '/'); // eslint-disable-line no-useless-escape
```

Comment:
Any reason to keep the useless escapes, assuming they are actually useless?

Reply:
Simplest explanation is I don't trust my RegEx enough to not somehow break it 😄